plugins/optimization-detective/storage/rest-api.php

```diff
@@ -346,3 +325,65 @@ function od_trigger_page_cache_invalidation( int $cache_purge_post_id ): void {
 	/** This action is documented in wp-includes/post.php. */
 	do_action( 'save_post', $post->ID, $post, /* $update */ true );
 }
+
+/**
```

Comment:
I just had a thought that perhaps an [`Accept`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) response header should be added for this endpoint. This apparently isn't needed because I don't see `Accept: application/json` being served anywhere now in REST API responses. Just something to think about.